Interfaces/RPPorts.py

```diff
@@ -206,25 +201,21 @@ def __pwd_out(self, channel, duration, dutycycle):
         sleep(duration/1000)    # to add a  delay in seconds
         pwm.stop()
 
-    def __pwm_out(self, channel, freq, duration, dutycycle=100, pulse_freq=0):
+    def __pwm_out(self, channel, freq, dutycycle=100, pulse_freq=0):
```

Comment:
rename this function to reflect the way we generate the stimulus : instead of pwm it is now pulse..